src/scheduler/server_info.c

```diff
@@ -709,7 +709,11 @@ query_server_dyn_res(server_info *sinfo)
 				}
 
 				if (set_resource(res, buf, RF_AVAIL) == 0) {
-					return -1;
+					snprintf(buf, sizeof(buf), "Script %s returned bad output",
```

Comment:
Do we need the capital S for script? Guess it is not a norm we follow.

Reply:
It's a caps for start of the log message sentence, I just followed the style of other log messages in the function:
snprintf(buf, sizeof(buf), "**C**an't pipe to program %s: %s",
		conf.dynamic_res[i].program, strerror(pipe_err));
else
snprintf(buf, sizeof(buf), "**E**rror piping to program %s.",
		conf.dynamic_res[i].program);

Reply:
Ok thanks for pointing out. I guess we do other things in other places..so it is fine.